src/gui_common/ChemicalEquation.cs

```diff
@@ -102,14 +100,19 @@ public bool ShowToggle
 
     public bool ProcessEnabled
     {
-        get => lastToggle;
+        get
+        {
+            if (EquationFromProcess == null)
+                return true;
+
+            return EquationFromProcess.Enabled;
+        }
         set
         {
-            if (value == lastToggle)
+            if (EquationFromProcess == null)
                 return;
```

Comment:
I think this should print an error message in case there's a programmer error in the future (so that it'll be quicker to diagnose).